Data/latest-version.txt

```diff
@@ -1 +1 @@
-4.0.2.3
+4.0.3.0
```

Comment:
4.0.2.4?

Reply:
That's fine, there's another patch that isn't released yet that is 4.0.2.3 which is fixing some install issues, so we can probably actually just keep 4.0.2.3